scripts/ci/router-check.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+set -u
+set -o pipefail
+
+# wait for the router
+#KUBECONFIG="${HOME}"/clusters/nested/auth/kubeconfig
+KUBECONFIG="${HOME}"/Documents/installer/somalley/auth/kubeconfig
```

Comment:
This will probably need to be updated before we land the PR ;).  I don't see anything in this PR that has opinions on how `router-check.sh` will be called, so maybe just drop `KUBECONFIG` (and the wait on it) from this script and leave waiting and setting the variable up to the caller?

Reply:
doh! yup